go.mod

```diff
@@ -177,4 +177,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace github.com/tmc/langchaingo => github.com/Abirdcfly/langchaingo v0.0.0-20240110125843-3feb6668f787 // branch arcadia
+replace github.com/tmc/langchaingo => github.com/Abirdcfly/langchaingo v0.0.0-20240124015404-c7798664fdb1 // branch arcadia
```

Comment:
Note where you have changed.

Reply:
And can we move back to the official repo ?

Reply:
This is a draft that is being edited.....

https://github.com/kubeagi/arcadia/pull/623#issue-2097279704